4-auth/config/settings.example.yml

```diff
@@ -1,11 +1,9 @@
 default: &default
+  project_id: your-project-id
+  storage_bucket: your-bucket-name
```

Comment:
storage_bucket is generic and may confuse readers, how about `gcs_bucket`? I'm looking for something for GCS centric.

Reply:
Changed, across all relevant files. Also changed in the tutorial text in the corresponding internal CL.